module/config.mjs

```diff
@@ -2602,6 +2602,27 @@ DND5E.spellScrollIds = {
 
 /* -------------------------------------------- */
 
+/**
+ * @typedef {object} SpellScrollValues
+ * @property {number} bonus  Attack to hit bonus.
+ * @property {number} dc     Saving throw DC.
+ */
+
+/**
+ * Spell scroll save DCs and attack bonus values based on level in the 2014 rules. If matching level isn't found,
```

Comment:
Let's assume these values will be the same in the 2024 rules too unless we see otherwise.

```suggestion
 * Spell scroll save DCs and attack bonus values based on level. If matching level isn't found,
```